README.md

```diff
@@ -1,9 +1,9 @@
+<<<<<<< HEAD
+Somebody please make this readme beautiful
+
+true
+=======
 #Team Space Platypi
```

Comment:
To revert this file, I would just copy and paste the raw file from develop into your branch and make changes until the READMe.md doesn't show up in this PR.